bin/build_package.ts

```diff
@@ -34,117 +33,105 @@ async function main() {
         process.exit(1);
     }
 
-    const packageData = await import(path.join(packageRoot, 'package.json'), {with: { type: 'json'}});
-    const packageName = packageData.name;
-    const srcDir = path.join(packageRoot, 'src');
-    const distDir = path.join(packageRoot, 'dist');
+    const packageData = await readPackageJSON(path.join(packageRoot, 'package.json'));
+    const isStimulusBundle = '@symfony/stimulus-bundle' === packageData.name;
+    const isReactOrVueOrSvelte = ['@symfony/ux-react', '@symfony/ux-vue', '@symfony/ux-svelte'].some(name => packageData.name.startsWith(name));
```

Comment:
```suggestion
    const isReactOrVueOrSvelte
```

hahaha i may have use a... another const name

Reply:
Yeah I wasn't really inspired here... 😅